src/shogun/labels/BinaryLabels.cpp

```diff
@@ -64,6 +65,22 @@ CBinaryLabels::CBinaryLabels(CFile * loader) : CDenseLabels(loader)
 {
 }
 
+CBinaryLabels::operator CMulticlassLabels()
+{
+	CMulticlassLabels src;
+	
+	for (index_t i = 0; i < m_labels.vlen; i++)
+	{
+		if(m_labels.vector[i] == -1)
+		{
```

Comment:
Ok, I will fix these style problems, but I saw some source code that use {} for single line commands in BinaryLabels.cpp, so I Imitate them.